src/w3c/defaults.js

```diff
@@ -32,6 +32,8 @@ const w3cDefaults = {
   logos: [],
   xref: true,
   wgId: "",
+  otherLinks: [],
+  excludeGithubLinks: true,
```

Comment:
Hidden by default? Could it be a ReSpec specific addition to keep it on always?

Reply:
That’s being added to fixup.js thankfully.